test/regression/sql/prepare.sql

```diff
@@ -0,0 +1,54 @@
+-- the view should return the empty
+SELECT name, statement, parameter_types FROM pg_prepared_statements;
+
+CREATE DATABASE testdb;
+CREATE TABLE copy_database(datname text, datistemplate boolean, datallowconn boolean);
+INSERT INTO copy_database SELECT datname, datistemplate, datallowconn FROM pg_database;
```

Comment:
What's the reason you're creating a new database for this query?

Reply:
I changed it to postgres now